AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1953,7 +1953,7 @@ nevasport.ru##.post-tags__yn
 nevasport.ru##.post-tags__gn
 nevasport.ru##.post-tags__yz
 arquidiocesedeteresina.org.br##.widget_sola_nl_subscribe_widget
-pressherald.com##.gform_wrapper
+infosecinstitute.com,pressherald.com##.gform_wrapper
```

Comment:
usually we do not block elements in footer. It is not annoying.

Reply:
@wout-codes please remove the domain